### ISSUE TITLE
Create a home page (CTB routing)

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#57]
### Who

- [x] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

What is the intended feature?

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
